lib_nbgl/include/nbgl_use_case.h

```diff
@@ -137,10 +137,67 @@ typedef struct {
     uint8_t nbContents;  ///< number of contents
 } nbgl_genericContents_t;
 
+/**
+ * @brief The different types of operation to review
+ *
+ */
+typedef enum {
+    TYPE_OPERATION = 0,
+    TYPE_TRANSACTION,
+    TYPE_MESSAGE,
+} nbgl_operationType_t;
+
+/**
+ * @brief The different types of review status
+ *
+ */
+typedef enum {
+    STATUS_TYPE_OPERATION_SIGNED = 0,
+    STATUS_TYPE_OPERATION_REJECTED,
+    STATUS_TYPE_TRANSACTION_SIGNED,
+    STATUS_TYPE_TRANSACTION_REJECTED,
+    STATUS_TYPE_MESSAGE_SIGNED,
+    STATUS_TYPE_MESSAGE_REJECTED,
+    STATUS_TYPE_ADDRESS_VERIFIED,
+    STATUS_TYPE_ADDRESS_REJECTED,
+} nbgl_reviewStatusType_t;
+
 /**********************
  * GLOBAL PROTOTYPES
  **********************/
 
+void nbgl_useCaseHomeAndSettings(
+    const char                *appName,
```

Comment:
Why don't you use directly the new structure `nbgl_homeAndSettingsContext_t`?

Reply:
Could be... but it makes using more and more structures...
I don't have an opinion on this...

Reply:
Done partially with action context